### DISCUSSION TITLE
how exactly do you use the download custom rooms feature?

### DISCUSSION THREAD

**Discussion by fluffykanade** [RealMCoded/Rec.js#7] [Q&A] [answered]
sorry for posting another q&a lol

**Comment by RealMCoded** [comment, marked as answer]
## Notice: Custom room loading is not functional yet.

There are 2 ways to use the custom room downloader. 

## Method 1: Easy Menu

The easy menu is more of a guided approach to downloading rooms. You can enter it by launching the app and selecting "Download Custom Room" running the command `npm rooms`

From there, it should prompt you to enter the name of the room you want to download. It's not case sensitive, so you could enter "reccenter", "RecCenter", or even "ReCcEnTeR". When it is finished downloading it will ask if you want to download another room.

## Method 2: Batch download

This method skips the easy menu and is a way faster way to download custom rooms.

You can't do this from the menu, you have to use the command. You want to just enter the names of the rooms you want to download as arguments. 

```
npm rooms <room 1> <room 2> <room 3> <etc...>
```

an example of that command would be
```
npm rooms reccenter gribbly.exe XmansDorm 1600_holotars
```
